package.json

```
@@ -30,7 +30,7 @@
 	},
 	"peerDependencies": {
 		"rollup": ">=2.0.0",
-		"svelte": ">=3.5.0"
+		"svelte": ">=3.5.0 || >=5.0.0-next.120"
```

I would imagine we should allow any pre-release version here
```suggestion
		"svelte": ">=3.5.0 || >=5.0.0-next.1"
```

---

I don't really have an opinion on it. I chose 120 to just peg it to the release candidate. I'm not sure how many people would be using the pre-release who wouldn't want to update.

---

Happy to commit this change if you want it.